mscr-ui/src/modules/form/metadata-form/index.tsx

```diff
@@ -267,6 +267,17 @@ export default function MetadataForm({
             </Grid>
           </MetadataRow>
 
+          <MetadataRow container>
+            <Grid item xs={4}>
+              <MetadataLabel>{t('metadata.soure-url')}</MetadataLabel>
```

Comment:
Typo in the word "source"

Reply:
okk will fix that and will merge it myself :)